float64 value
string node_name
---
